docker/install/ubuntu_install_caffe.sh

```diff
@@ -65,5 +65,4 @@ cd / && rm -rf /caffe_src
 
 PYCAFFE_ROOT=${CAFFE_HOME}/python
 echo "${CAFFE_HOME}/lib" >> /etc/ld.so.conf.d/caffe.conf && ldconfig
-VENV_SITE_PACKAGE=$(pip3 show numpy | grep "Location:" | cut -d ' ' -f 2)
-ln -s ${PYCAFFE_ROOT}/caffe ${VENV_SITE_PACKAGE}/caffe
```

Comment:
fyi #12813 fixed this too via site-packages

Reply:
true, do you have a preference? cc @ashutosh-arm

Reply:
oh sorry, reading the CI failure now. ignore me :)